lib/grape/locale/en.yml

```diff
@@ -33,4 +33,5 @@ en:
         at_least_one: 'are missing, at least one parameter must be provided'
         exactly_one: 'are missing, exactly one parameter must be provided'
         all_or_none: 'provide all or none of parameters'
+        invalid_group_type: 'group of required parameters must have a specified type of Array or Hash, type provided: %{group_type}'
```

Comment:
I don't think we want to have the type provided in the message, typically error messages should be fixed, and potentially additional data should indicate the source. I would make this just  `group of required parameters must be Array or Hash`.